src/Renderer/Shader/FeatureVS.glsl

```diff
@@ -0,0 +1,74 @@
+#version 100
+
+precision highp float;
+precision highp int;
+
+#define SHADER_NAME SimpleMaterial
+#define VERTEX_TEXTURES
+
+#define USE_LOGDEPTHBUF
+#define USE_LOGDEPTHBUF_EXT
+
+#ifdef USE_LOGDEPTHBUF
+
+    #define EPSILON 1e-6
+    #ifdef USE_LOGDEPTHBUF_EXT
+
+        varying float vFragDepth;
+
+    #endif
+
+    uniform float logDepthBufFC;
+
+#endif
+
+//attribute vec2      uv;
+attribute vec3      position;
+attribute vec3      normal;
+
+uniform mat4        projectionMatrix;
+uniform mat4        modelViewMatrix;
+
+uniform mat4       mVPMatRTC;
+uniform int        RTC;
+varying float      light;
+
+attribute vec3 color;
+varying vec3 vColor;
+
+// IE error : Initializer for const variable must initialize to a constant value
+//const vec3 dir =  normalize(vec3(1.0,1.0,0.5));
+
+void main()
```

Comment:
It's to transfer color information to the fragment shader.

I think I'll remove feature material and shaders from this PR before it's merged, they aren't really useful or relevant to this PR.